Formula/global.rb

```diff
@@ -35,7 +35,8 @@ class Global < Formula
   depends_on "python@3.9"
   depends_on "sqlite"
 
-  uses_from_macos "ncurses"
+  uses_from_macos "bison" => :build
+  uses_from_macos "ncurses"  
```

Comment:
```suggestion
  uses_from_macos "ncurses"
```
Oops. Sorry. Didn't see that in the GitHub UI.